mmedit/models/backbones/encoder_decoders/decoders/plain_decoder.py

```diff
@@ -77,5 +112,6 @@ def forward(self, inputs):
         out = self.max_unpool2d(out, max_idx_1)
 
         out = self.relu(self.deconv1_1(out))
+
```

Comment:
Is this line necessary?

Reply:
> Is this line necessary?

Sorry, it's my fault, I have retrieved this change.

Reply:
> Is this line necessary?

Sorry, it's my fault, I have retrieved this change.